yarn.lock

```diff
@@ -3,68 +3,602 @@
 
 
 "@babel/code-frame@^7.0.0-beta.35":
-  version "7.0.0-beta.37"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0-beta.37.tgz#2da1dd3b1b57bfdea777ddc378df7cd12fe40171"
+  version "7.0.0"
```

Comment:
Don't be alarmed by the number of lines changed here. The lockfile is only for development purposes, and I processed an upgrade for everything.